src/main/java/tech/gruppone/stalker/server/model/db/PlaceDao.java

```diff
@@ -0,0 +1,43 @@
+package tech.gruppone.stalker.server.model.db;
+
+import lombok.NonNull;
+import lombok.Value;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+
+@Value
+// no @table here because this corresponds to tables Place and PlaceData joined on id
+public class PlaceDao {
+
+  @Id
+  @Column("id")
+  long id;
+
+  @Column("organizationId")
+  long organizationId;
+
+  @NonNull
+  @Column("name")
+  String name;
+
+  @NonNull
+  @Column("positionGeoJson")
+  String rawPositionJson;
+
+  // we use a single DAO for tables Place and PlaceData
+  @NonNull
+  @Column("address")
+  String address;
+
+  @NonNull
+  @Column("city")
+  String city;
+
+  @NonNull
+  @Column("zipcode")
+  String zipcode;
+
+  @NonNull
+  @Column("state")
+  String state;
+}
```

Comment:
_rawPositionJson_ non è una colonna di PlaceData, si chiama **position** nella tabela Place in SQL
Per i campi restanti (address, city, zipcode, state) andrebbero in un file PlaceDataDao, ma se a te funziona e ci sono i loro motivi per averli inseriti in PlaceDao, allora può andare bene.

Reply:
purtroppo questo è l'unico modo che ho trovato per deserializzare il Polygon di MySQL che è un data type non mappato automaticamente da r2dbc. ho messo un nome diverso per rendere esplicita la cosa.

placedata l'ho messo qua dentro per evitare una query aggiuntiva al database (dato che c'è un rapporto 1 a 1 tra le tabelle place e placedata)

Reply:
segno risolto. probabilmente dovrò implementare anche placedatarepository ma farò in una pr separata